tools/create-devkit.ps1

```diff
@@ -32,7 +32,7 @@ New-Item -Path $DstPath\include -ItemType Directory > $null
 copy -Recurse inc\* $DstPath\include
 
 New-Item -Path $DstPath\lib -ItemType Directory > $null
-copy "artifacts\bin\$($Platform)_$($Config)\fnmpapi.lib" $DstPath\lib
```

Comment:
All these `$($Platform)_$($Config)` are going to have to be fixed too. In order to align with Windows, we build in the `amd64chk` format/style.

Reply:
Is that something that can be renamed into Windows-isms during vpack creation? For external, undocked, and open source projects, our default naming should be simpler and intuitive,

Reply:
No, it's not currently supported by the `microsoft/undocked` project. Both the build props and yml must be in sync, and your project file contents don't change when built internally. So, it would definitely complicate things to support multiple models and dynamically figure out which is being used.

Reply:
Ok, fair enough